src/core/utils/components/Popover.tsx

```diff
@@ -61,13 +61,14 @@ export const Popover = React.forwardRef((props: PopoverProps, ref) => {
   const computedProps: Partial<TippyProps> = {
     allowHTML: true,
     animation: false,
-    appendTo: 'parent',
+    appendTo: (ref) => ref.ownerDocument.body,
```

Comment:
Maybe call it `el` instead of `ref`?

`ref` implies that it's a ref object, which it isn't. This is more like `ref.current`.

Reply:
yeah, took ref from their definition

![image](https://user-images.githubusercontent.com/81580355/154457738-acc061b5-6ecf-41d2-a69d-a9011ca13430.png)

changed